src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -103,7 +108,15 @@ public DurableTaskExtension()
             IDurableHttpMessageHandlerFactory durableHttpMessageHandlerFactory = null,
             ILifeCycleNotificationHelper lifeCycleNotificationHelper = null,
             IMessageSerializerSettingsFactory messageSerializerSettingsFactory = null,
-            IErrorSerializerSettingsFactory errorSerializerSettingsFactory = null)
+            IErrorSerializerSettingsFactory errorSerializerSettingsFactory = null
+#if !FUNCTIONS_V1
+#pragma warning disable SA1113, SA1001, SA1115
+            ,ITelemetryActivator telemetryActivator = null

```

Comment:
NIT: I would rather duplicate the line of `IErrorSerializerSettingsFactory errorSerializerSettingsFactory = null` with a `,` and ` )` then to do all of the warning disablement, and have the weird comma and paranthesis placement. It forces a bunch of `#` statements that really clutter up the code.

Reply:
Done